app/gui/src/project-view/components/GraphEditor/widgets/WidgetSelection/SelectionSubmenu.vue

```diff
@@ -114,7 +115,7 @@ export interface SubmenuComponent {
       <SizeTransition height :duration="100">
         <DropdownWidget
           v-if="props.show"
-          :class="{ ExtendUpwards: props.topLevel }"
+          :class="{ ExtendUpwards: props.topLevel && props.extendUpwards !== false }"
```

Comment:
I think this approach to making `true` the default is a bit subtle--defining an explicit default using props destructuring or `withDefaults` would be clearer.